workitem/enum_type.go

```diff
@@ -5,12 +5,29 @@ import (
 	"reflect"
 
 	"github.com/fabric8-services/fabric8-wit/convert"
+	errs "github.com/pkg/errors"
 )
 
 type EnumType struct {
-	SimpleType
-	BaseType SimpleType
-	Values   []interface{}
+	SimpleType       `json:"simple_type"`
+	BaseType         SimpleType    `json:"base_type"`
+	Values           []interface{} `json:"values"`
+	RewritableValues bool          `json:"rewritable_values"`
+}
+
+// Ensure EnumType implements the FieldType interface
+var _ FieldType = EnumType{}
+var _ FieldType = (*EnumType)(nil)
+
+// DefaultValue implementes FieldType
+func (t EnumType) DefaultValue(value interface{}) (interface{}, error) {
```

Comment:
Some unit test would be good for this function.

Reply:
Yes, on it.

Reply:
@baijum, I've added tests in ea808de.